flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/EICAS_Common.js

```diff
@@ -67,6 +73,27 @@ class EICASCommonDisplay extends Airliners.EICASTemplateElement {
         }
         //}
     }
+    refreshISA(_value, _force = false) {
+        //if ((_value != this.currentISA) || _force) {
```

Comment:
```suggestion
```
I'm not sure why we still have this in the other functions. I would suggest to not add it back.

Reply:
Works for me. In my opinion, we should also remove it from the other functions then, including the `_force` parameter. It looks like the commented out lines have been there since implementation. It's unlikely someone is gonna want to uncomment them again. If it turns out this logic is required again for some reason, it's easy enough to reimplement. Would you agree?

Reply:
Actually, thinking about it now. I think we should put the check for `_value != this.currentISA` back in for everything (TAT, SAT, ISA) to avoid unnecessary DOM updates, but leave out the `_force` parameter until there's a need for it.

Reply:
@BlueberryKing I already did that in a branch, and nothing broke. So you can re-add it safely.

Reply:
@Benjozork Are you talking about putting the if statement back in? Because I ran into an issue where, once the ADIRS aligned, I would still see "XX" but in green for SAT and TAT because the `this.currentSAT` had not changed under the hood, so with this code, there was no need to rewrite the actual temperature values to the DOM until the temperature actually changes. We would at least need to force an update once the ADIRS aligned, if I identified correctly.

In my view, there are two options: We leave the `if` commented out, updating the DOM every time. This is not optimized, but we don't need to change any code and the QA is still valid. Or I could implement some optimizations to only update the DOM when needed, which would probably need to be retested. I don't mind which option we choose.

Reply:
I'd go with the second one. Actually I just remembered this but and I had to switch to tracking the entire text and not just the temperature value.

Reply:
Sounds like a good solution. At this point, I will also make sure that `refreshADIRS()` doesn't trigger an update every frame.